ldes-fragmentisers/ldes-fragmentisers-common/src/test/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/FragmentDeletionSteps.java

```diff
@@ -0,0 +1,77 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fragmentation;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.LdesFragmentIdentifier;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.TreeRelation;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entities.Fragment;
+import io.cucumber.java.DataTableType;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static be.vlaanderen.informatievlaanderen.ldes.server.domain.constants.RdfConstants.GENERIC_TREE_RELATION;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@SuppressWarnings("java:S3415")
```

Comment:
Why do we suppress this warning? Isn't this about swapping two arguments in an assertion?

Reply:
`org.junit.Assert.assertTrue(true)` did not give a warning, but `org.assertj.core.api.Assertions.assertThat(true).isTrue()` does give a warning, as the AssertJ library thinks it is odd to pass a hardcoded boolean value in `assertThat()`

Reply:
ok, I am fine with the format "assertThat(true).isTrue()", did not know that sonar had issues with this.. Then you can keep the suppress and ignore my comments :+1: